tests/unit/test_equilibrium_single.py

```diff
@@ -0,0 +1,498 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+from pytest import approx
+from pytest import mark
+from scipy.integrate import quad
+
+import xarray as xr
+from xarray import broadcast
+from xarray import DataArray
+from xarray import where
+
+from indica.converters import FluxSurfaceCoordinates
+from indica.converters import TransectCoordinates
+from indica.converters import TrivialTransform
+from indica.converters import LinesOfSightTransform
+from indica.equilibrium import Equilibrium
+
+from scipy.optimize import fmin
+
```

Comment:
You probably don't need to make all of these routines as general as I did for Property Based Testing. You can just have them generate one, simple, type of data. You might find that you don't need `smooth_funcs` at all anymore, for example. (I'll leave a comment with a more concrete example below). It looks like you've already done a lot of the work on this, so if things work then you might as well just leave it as is, but this could save time in future.